### PR TITLE
[TCSACR-596][NUI] Add missing privilege comments

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Add missing privilege comments

### API Changes ###
https://jira.sec.samsung.net/browse/TCSACR-596

### CODE REVIEW

**Comment by dongsug-song** [comment]
@wiertel 
Hello,
this patch shows "ACR Required" tag. 
I have just added <privilage> tag and no interface has been changed.
is it OK to be merged after accepted?
please confirm it.
thank you.

**Comment by wiertel** [comment]
Hello, I agree with the bot. The signatures do no change but the whole API contract is changed a little. Please submit a simple ACR.

**Comment by myroot** [comment]
http://tizen.org/privilege/display
> Allows the application to manage display settings, such as the brightness. This may increase battery consumption.
Type: Native / Dotnet
Level: Public
Privacy: No

왜 NUIApplication생성에 필요한 privilege가 될까요? 필요 없어보이고, 실제로 앱에서 없어도 동작에 문제가 없는데 왜 추가되는 걸까요?

**Comment by dongsug-song** [comment]
> http://tizen.org/privilege/display
> 
> > Allows the application to manage display settings, such as the brightness. This may increase battery consumption.
> > Type: Native / Dotnet
> > Level: Public
> > Privacy: No
> 
> 왜 NUIApplication생성에 필요한 privilege가 될까요? 필요 없어보이고, 실제로 앱에서 없어도 동작에 문제가 없는데 왜 추가되는 걸까요?

안녕하세요.
확인 감사드립니다.
네. dali 파트에 문의했고 필요 없는것으로 답변 받아서, 제거 하였습니다.

File: src/Tizen.NUI/src/public/Window/Window.cs [hunk 1]
```diff
@@ -869,6 +869,8 @@ public void SetInputRegion(Rectangle inputRegion)
         /// </summary>
         /// <param name="level">The notification window level.</param>
         /// <returns>True if no error occurred, false otherwise.</returns>
+        /// <privilege>http://tizen.org/privilege/window.priority.set</privilege>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/Window/Window.cs:873]: "can be due to..." -> "can be thrown due to..." will sound better, if you decide to change it then change all occurrences

Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Window/Window.cs]: @wiertel 
Hello, thank you for the review.
I have changed by following your comment.
please review again. thank you.

**Review by Seoyeon2Kim — APPROVED** [review]
lgtm

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Review by hinohie — APPROVED** [review]
ACR Accepted.

**Pull request merged**